sandbox/simulator/randomtext

```diff
@@ -45,23 +45,37 @@ function GenerateRandomPostTitle()
 // Future: something much more meaningful. Also figure out how to include image uploads, links, other fun features.
 function GenerateRandomPostText($maxParagraphs = 3)
 {
+	$output = "";
+
+	// Add text to the post
 	global $usewikitext;
 	if($usewikitext)
 	{
 		$paragraphs = random_int(1,$maxParagraphs);
-		$output = "";
 		for($p =0;$p<$paragraphs;$p++)
 		{
 			$sentences = random_int(1,10);
 			for($s=0;$s<$sentences;$s++)
 			{
 				$output .= GenerateRandomPhrase(3,17) . ". ";
 			}
-			$output .= "\n\n";
+			$output .= "\n";
 		}
-		return $output;
 	}
 
+
+	// Add youtube video to the post
+	global $useyoutubevideos;
+	global $youtubevideochance;
+	if ($useyoutubevideos and random_int(0, $youtubevideochance) == 0)
```

Comment:
As coded, this is a 1 in ($...chance + 1) probability - random_int is inclusive on both ends